.github/workflows/tf-account.yaml

```diff
@@ -12,12 +12,25 @@ on:
         required: true
       TF_BACKEND_S3_REGION:
         required: true
+      TS_OAUTH_CLIENT_ID:
+        required: false
+      TS_OAUTH_SECRET:
+        required: false
     inputs:
       terraform-version:
         description: 'Terraform version'
         required: false
         type: string
         default: '1.5.4'
+      TS_TAGS:
```

Comment:
Inputs should be `dash-case`.

Reply:
More importantly, inputs are an abstraction layer, so avoid abbreviations:

`TS_TAGS` -> `tailscale-tags`